overwolf/src/FrameMenu.tsx

```diff
@@ -108,6 +122,14 @@ const useStyles = makeStyles()(theme => ({
             background: 'rgba(255, 255, 255, 0.33)',
         },
     },
+    iconCategory: {
+        display: 'flex',
+        alignItems: 'center',
+
+        '& > label': {
+            flexGrow: 1,
```

Comment:
Could you add a single element (maybe `<span />`) after the label element in iconCategory summaries, and target that with `flexGrow: 1`? With this change, the label will fill the `summary`, and clicking the empty space will then toggle the checkbox, not the expand/collapse the `details`.